buildSrc/src/main/kotlin/profiler.android-studio-setup.gradle.kts

```diff
@@ -76,26 +94,46 @@ abstract class AndroidStudioInstallation {
     abstract val studioInstallLocation: DirectoryProperty
 }
 
+abstract class AndroidSdkInstallation {
+    @get:InputFiles
+    @get:PathSensitive(PathSensitivity.RELATIVE)
+    abstract val sdkInstallLocation: DirectoryProperty
```

Comment:
That means the SDK directory would be an input. I am not sure snapshotting this directory makes much sense.

Reply:
removed